### DISCUSSION TITLE
Removal of approvers from CODEOWNERS?

### DISCUSSION THREAD

**Discussion by anuraaga** [open-telemetry/opentelemetry-collector-contrib#2736] [General]
I notice that I have a hard time sorting through notifications for this repo because currently all PRs have a review request from the approvers team

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L15

We already have the auto_assign script to balance out load among approvers - would it make sense to remove the line in CODEOWNERS? Currently I look at all assigned PRs, except dependabot, as the sort of assigned load, but I also need to keep a tab on AWS-related PRs for which there are a few subfolder CODEOWNERS assigned to me. But because every PR in the repo comes in with `review requested` it's very easy to miss AWS-related PRs due to the sheer volume. I could be missing a corner case though, but if it makes sense removing the global CODEOWNERS could be helpful. What do you think @bogdandrutu @tigrannajaryan?

**Comment by tigrannajaryan** [comment]
It was discussed in the past and we decided not to do it. @bogdandrutu can provide more details (I don't remember what exactly was the problem).
